lib/mailer.js

```diff
@@ -99,7 +102,9 @@ module.exports = function (config, log) {
             uaBrowserVersion: opts.uaBrowserVersion,
             uaOS: opts.uaOS,
             uaOSVersion: opts.uaOSVersion,
-            timestamp: opts.timestamp
```

Comment:
@jrgm Just replacing timestamp with a timezone, which will allow the auth-mailer to derive the correct timestamp (formatted to suit the locale as well).